manim/utils/ipython_magic.py

```diff
@@ -137,12 +137,5 @@ def construct(self):
                 )
 
 
-def _video_hash(path):
-    sha1 = hashlib.sha1()
-    with open(path, "rb") as f:
-        while True:
-            data = f.read(65536)
-            if not data:
-                break
-            sha1.update(data)
-    return sha1.hexdigest()
+def _generate_file_name():
```

Comment:
Can you add a test for this function?

Reply:
I am not sure, as this makes only an output file name, do we really need a test for this?

Reply:
A small test checking whether the returned from this function can be created.
as in we shouldn't add here something which will not work on Windows...

Reply:
I added a test but maybe not what you were looking for, I did one to check the format with regex. I am working on another jupyter related feature and was trying to add tests for it, there is a problem though when testing for file output. 

```py
local_path = Path(config["output_file"]).relative_to(Path.cwd())
```

This line throws an error when I try to run with a tmp_path:

```py
ip.run_line_magic(
        "manim", "-qm -v WARNING -s --media_dir " + tmp_path + " SquareToCircle"
)
```

I thought one solution was to check if it is an absolute path, but absolute paths won't work for jupyter so we would only be changing it for tests